### PR TITLE
Update GYP

### PR DESCRIPTION
**Pull request by zdne**
The new version of GYP is needed for the support of the actual Microsoft Visual Studio compilers.

### CODE REVIEW

File: tools/README.md [hunk 1]
```diff
@@ -1,22 +1,18 @@
 # Snow Crash Development Tools
 
 ## Gyp
-Snow Crash uses (bundled) [Gyp](http://code.google.com/p/gyp/) build automation tool to generates its Makefiles. 
+Snow Crash uses (bundled) [Gyp](https://gyp.gsrc.io/) build automation tool to
+generate its Makefiles.
 
 ### Actual Bundled Version
-**r1846**
+[**SHA d84fed246cd42612c31a56a897e2d032e254a58a**](https://chromium.googlesource.com/external/gyp/+/d84fed246cd42612c31a56a897e2d032e254a58a)
```
Comment (kylef) [inline tools/README.md]: Now that Gyp uses Git. Why not submodule it instead?

File: tools/README.md [hunk 2]
```diff
@@ -1,22 +1,18 @@
...
-	
-	```sh
-	$ svn checkout http://gyp.googlecode.com/svn/trunk/ gyp
-	```
+1. Download the latest stable version of Gyp from its [repository](https://chromium.googlesource.com/external/gyp/+/refs/heads/master)
 
 2. Remove unnecessary files:
 
 	```sh
-	$ cd gyp
```
Comment (kylef) [inline tools/README.md]: No longer need to remove `.svn` because it's not SVN.

File: tools/README.md (hunk 1 above)
Comment (w-vi) [inline tools/README.md]: I don't like submodules with no control over the submodule repo. I think this is good enough.

Comment (kylef) [inline tools/README.md]: Well, we can have our own fork like we do for [sundown](http://github.com/apiaryio/sundown). Using a submodule would make it easier to update and keep track of history better.

Comment (w-vi) [inline tools/README.md]: If we make a fork, then I am in favor of using submodule.

Comment (pksunkara) [inline tools/README.md]: > I don't like submodules with no control over the submodule repo

@w-vi Why is that? I am just wondering about your view on this.

Comment (w-vi) [inline tools/README.md]: What if the commit we are referring in that repo we have no control of is removed? What if somebody messes up the master?

**Pull request merged**